apis/v1alpha1/validation_test.go

```diff
@@ -111,6 +193,11 @@ func TestValidateResouceMapping(t *testing.T) {
 			if diff := testutil.DiffErrString(err, tc.expErr); diff != "" {
 				t.Errorf("ValidateResourceMapping got unexpected error: %s", diff)
 			}
+			if tc.wantSubscope != "" {
+				if diff := cmp.Diff(tc.data.Resource.Subscope, tc.wantSubscope); diff != "" {
```

Comment:
Otherwise you output would be wrong
```suggestion
				if diff := cmp.Diff(tc.wantSubscope, tc.data.Resource.Subscope); diff != "" {
```